### PR TITLE
Replace booster by example

### PR DESCRIPTION
**Pull request by aureamunoz**
Related to: SB-1345

### CODE REVIEW

File: lib/constants.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 export const SNOWDROP_ORG = "snowdrop";
 
-export const BOOSTER_GITHUB_TOPIC = "booster";
+export const BOOSTER_GITHUB_TOPIC = "example";
```
Comment (geoand) [inline lib/constants.ts]: I am pretty sure we also need to change: https://github.com/snowdrop/snowdrop-automation-client/blob/b4e3fa574cb3dea7a813e032670a37871cd29d51/lib/constants.ts#L14

Comment (geoand) [reply lib/constants.ts]: Which is why the tests are failing it seems

Comment (aureamunoz) [reply lib/constants.ts]: Yes, I have modified it

Comment (geoand) [reply lib/constants.ts]: Yup, should be good!

**Comment by gytis** [comment]
Not at the moment. But at some point in the future it would be good to change variable names, methods, directories etc.

**Pull request merged**